File: HorseRacing.Contracts/Models/Game/Responses/GetAvailableSuit/GetAvailableSuitResponse.cs
using HorseRacing.Contracts.Base.Responses;

namespace HorseRacing.Contracts.Models.Game.Responses.GetAvailableSuit
{
    public class GetAvailableSuitResponse : BaseListResponse<GetAvailableSuitResponseDto>
    {
        public GetAvailableSuitResponse() { }
        public GetAvailableSuitResponse(List<GetAvailableSuitResponseDto> data) : base(data) { }
    }
}

File: HorseRacing.Web/src/interfaces/api/contracts/model/game/responses/get-available-suit/get-available-suit-response.ts
import type { BaseListResponse } from "~/interfaces/api/contracts/base/responses/base-response";
import type { GetAvailableSuitResponseDto } from "./get-available-suit-response-dto";

export class GetAvailableSuitResponse implements BaseListResponse<GetAvailableSuitResponseDto> {
    DataValues: GetAvailableSuitResponseDto[];

    constructor(data?: GetAvailableSuitResponseDto[]) {
        this.DataValues = data || [];
    }
}
